File: src/main/scala-graphql/io/github/graphql/resolver/LabelableResolver.scala
package io.github.graphql.resolver

import io.github.graphql.model._

/**
 * Resolver for Labelable
 */
@javax.annotation.Generated(
    value = Array("com.kobylynskyi.graphql.codegen.GraphQLCodegen"),
    date = "2021-03-06T19:48:01+0800"
)
trait LabelableResolver {

    @throws[Exception]
    def labels(labelableTO: LabelableTO, after: String, before: String, first: Option[Int], last: Option[Int]): LabelConnectionTO

}

File: src/main/kotlin-graphql/io/github/graphql/resolver/LabelableResolver.kt
package io.github.graphql.resolver

import io.github.graphql.model.*

/**
 * Resolver for Labelable
 */
@javax.annotation.Generated(
    value = ["com.kobylynskyi.graphql.codegen.GraphQLCodegen"],
    date = "2021-03-06T19:48:04+0800"
)
interface LabelableResolver {

    @Throws(Exception::class)
    fun labels(labelableTO: LabelableTO, after: String?, before: String?, first: Int?, last: Int?): LabelConnectionTO?

}
